src/app/api/quality_matrix/replication_source.py

```diff
@@ -119,42 +119,44 @@ def missing_fields(
     return {search_keyword: missing_fields_ratio(value, total_count)}
 
 
-async def store_in_timeline(
-    data: QUALITY_MATRIX_RETURN_TYPE, database: Database, form: Forms
-):
-    await database.connect()
-    await database.execute(
-        sqlalchemy.insert(Timeline).values(
-            {
-                "timestamp": datetime.now().timestamp(),
-                "quality_matrix": data,
-                "form": form,
-            }
-        )
-    )
-
-
 async def items_in_response(response: Response) -> dict:
     return response.aggregations.to_dict().items()
 
 
-REPLICATION_SOURCE_ID = "ccm:replicationsource"
 PROPERTIES = "properties"
```

Comment:
No longer used here - but probably imported from somewhere else. Eventually move it to where it is imported (if it is only one location)

Reply:
PROPERTIES is used here, e.g. in create_properties_search. I will leave it here for now.